doc/crypto/api/ops/signature.rst

```diff
@@ -465,16 +465,13 @@ EdDSA signature algorithms
 
```

Comment:
The new functions should probably be added/mentioned in the "Signature functions" subsection in the chapter pre-amble.